src/EnergyPlus/ScheduleManager.hh

```diff
@@ -517,16 +513,26 @@ struct ScheduleManagerData : BaseGlobalStruct
     std::map<std::string, int> dayScheduleMap;
     std::map<std::string, int> weekScheduleMap;
 
+    Sched::ScheduleConstant SchedAlwaysOff;
+    Sched::ScheduleConstant SchedAlwaysOn;
+
     void init_constant_state(EnergyPlusData &state) override
     {
-        Sched::InitConstantScheduleData(state);
+        InitConstantScheduleData();
+        AddConstantSchedulesToContainers();
```

Comment:
In init_constant_state, I will set the SchedAlwaysOff/On members.
Then I add them to schedules and schedulesMap